lib/discourse_translator/translator_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module DiscourseTranslator
+  module TranslatorHelper
+    def self.translated_value(original_value, model, scope)
+      return original_value if !SiteSetting.experimental_topic_translation
+      return original_value if scope.request.params["show"] == "original"
+
+      translated = model.custom_fields[TRANSLATED_CUSTOM_FIELD]
+      return original_value if (translated.blank? || translated[I18n.locale].blank?)
+
+      translated[I18n.locale]
```

Comment:
I'm not sure if we are ready to solve this yet but if the translation is missing, do we display any indication in the interface for the user? It would be odd to the user if they request for a translation and we return the original one.